src/components/crownConfig.vue

```diff
@@ -88,9 +88,10 @@ export default {
   },
   computed: {
     style() {
-      const { x, y } = this.shape.findView(this.paper).getBBox();
+      const { x, y, width: shapeWidth } = this.shape.findView(this.paper).getBBox();
       const width = (this.node.diagram.bounds.width * this.paper.scale().sx);
-      return 'top:' + (y - 45) + 'px;left:' + (x + width - 20) + 'px;cursor:pointer';
+      const widthOffsetFromLabel = shapeWidth - width > 3 ? (shapeWidth / 2) - 15 : 0;
```

Comment:
Ugh. The label moves the x to the left when dragging a boundary event. This calculation got it as close to the original placement as I could get. Any way to get just the shape?

Reply:
@lisa-fehr You can get the width from `this.shape`:
```js
const shapeWidth = this.shape.size().width;
```

That should return the original size even if the label extends outside the shape bounds.

Reply:
The width is fine. It is the x/y that is including the label and moving it to the left of the shape.